crates/ruff/src/rules/flake8_todo/snapshots/ruff__rules__flake8_todo__tests__missing-author-in-todo_TDO002.py.snap

```diff
@@ -0,0 +1,12 @@
+---
+source: crates/ruff/src/rules/flake8_todo/mod.rs
+---
+TDO002.py:5:1: TDO002 Missing author in TODO
```

Comment:
Should we extend the message to explain where the author is expected or give an example?